lib/ui/password_entry_page.dart

```diff
@@ -478,6 +478,7 @@ class _PasswordEntryPageState extends State<PasswordEntryPage> {
             showAppBar: false,
             isDismissible: false,
             onDone: onDone,
+            showProgressBar: true,
```

Comment:
This page is also used for password reset/update. Do we want to set this true for all variants?